dsgrid_project/dimension_mappings/base_to_supplemental/electric_end_uses_noop.csv

```diff
@@ -1,41 +1,44 @@
 from_id,to_id
```

Comment:
Seems like the "base_to_supplemental" folder isn't really needed, since that's the only type of dimension_mapping generally associated with projects?

Reply:
What about #221?

Reply:
Good reminder--yes, that would be nice! Oh, and I see that that could be a different type of dimension mapping ...